File: 3194-find-words-containing-character/3194-find-words-containing-character.py
class Solution(object):
    def findWordsContaining(self, words, x):
        """for i in range()
        :type words: List[str]
        :type x: str
        :rtype: List[int]
        """
        res=[]
        for i in range(0,len(words)):
            if(words[i].count(x)>0):
                    res.append(i)
        return res 

File: 3194-find-words-containing-character/3194-find-words-containing-character.rb
# @param {String[]} words
# @param {Character} x
# @return {Integer[]}
def find_words_containing(words, x)
res=[]
for i in 0...words.length
     for j in 0...words[i].length
        if words[i][j]==x
         res<<i
         break
        end
    end 
 end 
    res
end